doc/sphinx/conf.py

```diff
@@ -188,11 +188,12 @@ def setup(app):
     'dirpath',
     'modpath',
     'module',
-    'red_expr',
+    'simple_tactic',
     'symbol',
     'tactic',
     'term_pattern',
     'term_pattern_string',
+    'toplevel_selector',
```

Comment:
It is for one of the 3 other rst files that I'm already updating.  It's easier to update orderedGrammar and common.edit_mlg for all of them in this PR rather than merge them later.  And if I'm doing that, I might as well do it for conf.py as well.

Reply:
I see, but I wish us to try not to expand this list. This is usually possible for me to do some suggestions working around that when reviewing your changes. So, if this isn't too much of a hassle, I'd like to revert this change before merging.

Reply:
When we've plugged in all the grammar, these will all go away.  That's going to happen pretty quickly, right?  The user impact is nil; `toplevel_selector` is still defined (probably incorrectly) and `simple_tactic` doesn't exist in this version.  I think it's a bit of wasted motion to revert it and then restore it later.  There are bigger fish to fry.

What do you think?

Reply:
I won't bother you with this. It's just easier to track that `toplevel_selector` is still defined if it doesn't appear in this list. In the next PR, if `toplevel_selector` is removed, it's easier to me to notice if it is added to this list.